zinit.zsh

```diff
@@ -19,11 +19,10 @@ unset ZPLGM
 #
 # Common needed values.
 #
+0="${${(M)${0::=${(%):-%x}}:#/*}:-$PWD/$0}"
+ZINIT[ZERO]="$0"
```

Comment:
Why is this not added to the other files?

Reply:
It is added, namely the 0=… line is added to all zinit sources.

Reply:
It is added, namely the 0=… line is added to all zinit sources.